.github/workflows/windows.yml

```diff
@@ -26,7 +26,12 @@ jobs:
     - uses: actions/checkout@v2
       with:
         fetch-depth: 0
-    
+        
+    - name: Set up Python3
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.6'
+
```

Comment:
this is to avoid that python2 issue?

Reply:
Yes. Python2 removal coming right after (wip)